### DISCUSSION TITLE
WinterCMS Support?

### DISCUSSION THREAD

**Discussion by helmutkaufmann** [skydiver/october-plugin-forms#251] [General]
Any intention to port/support this plugin on WinterCMS (https://wintercms.com)?

**Comment by skydiver** [comment]
Absolutely ... planning to release v2.0 for WinterCMS only
Didn't have time to take a look how to "migrate", but it's on my list

**Reply by helmutkaufmann** [reply]
Great. Migration would be painless at the moment. Just change the composer.json (in fact, it still works with the oc- prefix). That would be it.
